cdap-proto/src/test/java/co/cask/cdap/proto/codec/NamespacedIdCodecTest.java

```diff
@@ -40,12 +40,12 @@
   private final Id.Namespace ns = Id.Namespace.from("ns");
   private final Id.Application app = Id.Application.from(ns, "app");
   private final Id.Program program = Id.Program.from(app, ProgramType.CUSTOM_ACTION, "action");
-  private final Id.Flow flow = Id.Flow.from(app, "flow");
```

Comment:
Currently it's possible to have an `Id.Program` and `Id.Flow` that represents the same thing, but  it's better to only have `Id.Program` since it's less confusing.